RELEASE_NOTES.md

```diff
@@ -6,64 +6,31 @@
 
 Release Procedure is defined in the [CONTRIBUTING](CONTRIBUTING.md#release-procedure) document.
 
-## v3.3.0
+## v4.0.0
 
-- For the mainnet, this release includes update from v3.1.x → v3.3.x. Please also look at the [`v3.2` Release Notes](https://github.com/umee-network/umee/blob/v3.2.0/RELEASE_NOTES.md), notably the **Gravity Bridge Slashing**.
-- For the Canon-2 testnet, this release includes update from v3.2.x → v3.3.x
+This release contains the Historacle Upgrade, a pricing update which improves the way we treat quickly-changing prices in the leverage module.
 
-v3.2.0 was not released on mainnet due to a bug in x/leverage gov messages migration to the new format which utilizes x/gov/v1 authorization system. The bug caused legacy token registry updates to break x/gov proposal queries. In v3.3 we fix that bug.
+* See the [Historacle Design Doc](/docs/design_docs/011-historacle-pricing.md) for a description of how these prices are calculated.
+* See the [Leverage Module Spec](/x/leverage/README.md#historic-borrow-limit-value) for a description of how these prices are treated by the leverage protocol.
 
-Additional highlights:
-
-- Added `QueryMaxWithdraw` and `MsgMaxWithdraw` to allow user easily withdraw previously supplied tokens from the module back to the user balance.
-- Updated Cosmos SDK to v0.46.7
-
-Please see the [CHANGELOG](https://github.com/umee-network/umee/blob/v3.3.0/CHANGELOG.md) for an exhaustive list of changes.
-
-### Github Release
```

Comment:
let's keep the `Github Release` section. Better to be explicit and repeat informing users about the upgrade procedure, than going back and forth on a chat.

Reply:
f6cfe968cebd905d5ca1661832ea182d9d9bb05d